tests/lib/cli-engine.js

```diff
@@ -53,21 +54,30 @@ describe("CLIEngine", function() {
     function getFixturePath() {
         var args = Array.prototype.slice.call(arguments);
         args.unshift(fixtureDir);
-        return path.join.apply(path, args);
+        var filepath = path.join.apply(path, args);
+        try {
+            filepath = fs.realpathSync(filepath);
+            return filepath;
+        } catch (e) {
+            return filepath;
+        }
     }
 
     // copy into clean area so as not to get "infected" by this project's .eslintrc files
     before(function() {
-        fixtureDir = os.tmpdir() + "/eslint/fixtures";
+        fixtureDir = path.join(os.tmpdir(), "/eslint/fixtures");
         mkdir("-p", fixtureDir);
         cp("-r", "./tests/fixtures/.", fixtureDir);
+        fixtureDir = fs.realpathSync(fixtureDir);
     });
 
     beforeEach(function() {
+        process.chdir(path.join(fixtureDir, ".."));
```

Comment:
Just meant that we rely on this `cwd` option if we set it instead of changing the process work directory. If that feature breaks, all the tests will also fail. Do you expect any disadvantage from changing the work directory like done here?